### ISSUE TITLE
Latest version of Notepad++ incompatible

### ISSUE THREAD

**Issue by TechAngel85** [blu3mania/npp-papyrus#32]
Just want the title says, the latest version of this plugin is no longer working with Notepad++. The DLL fails.

**Comment by blu3mania** [comment]
Please be more specific about what version you are using and what exactly is "no longer working". I tried both NPP 8.4.2 (the latest version without safety delay) and 8.4.4 (the latest version) and as far as I can tell, all features of this plugin works fine.

**Comment by TechAngel85** [comment]
> 
![Screenshot 2022-07-17 124008](https://user-images.githubusercontent.com/15992075/179418001-651ab07a-88d4-45b5-a40b-1913c8998a83.png)

This is what I'm presented with every time I open Notepad++ unless I remove this plugin.

**Comment by blu3mania** [comment]
Likely because you are using an older version of this plugin. Please remove whatever version you are using in Plugin Admin, then reinstall the 0.4.0 version and you should be fine.

**Comment by TechAngel85** [comment]
Interesting. That wasn't it, but it lead to be find out that NPP somehow lost it's permissions to its own folder. Plugin Admin wasn't able to install updates or uninstall any of the plugins. It also wasn't actually removing the folder from the directory when I clicked Yes. I uninstalled and reinstalled NPP, installed all my plugins again and now things seem to be working as expected.

Sorry for the trouble, but thanks for the support!
